### PR TITLE
Task 201765 get attendee, and event details

### PR DESCRIPTION
**Pull request by yogesh-pawar-2023**
if event created for this user then get all details about event and attendee

https://tracker.tekdi.net/issues/201765?tab=time_entries

### CODE REVIEW

File: src/src/user/user.service.ts [hunk 1]
```diff
@@ -752,6 +752,34 @@ export class UserService {
 			  provider
 			  path
 			}
+		  }
+		    attendances {
```
Comment (parthgandhitekdi) [inline src/src/user/user.service.ts:756]: Please format document properly

Comment (yogesh-pawar-2023) [reply src/src/user/user.service.ts:756]: formated

**Review by parthgandhitekdi — APPROVED** [review]

File: src/src/user/user.service.ts [hunk 2]
```diff
@@ -753,6 +753,33 @@ export class UserService {
 			  path
 			}
 		  }
+		  attendances {
```
Comment (manojLondhe) [inline src/src/user/user.service.ts:756]: attendance or attendances?

Comment (yogesh-pawar-2023) [reply src/src/user/user.service.ts:756]: i have created  attendances  relationship

Comment (manojLondhe) [reply src/src/user/user.service.ts:756]: why do we need attendance in regular find user by id method?

**Review by manojLondhe — CHANGES_REQUESTED** [review]

**Pull request closed without merging**